### PR TITLE
Update documentation of Mongo configuration

### PR DESCRIPTION
**Pull request by guillaumebert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,32 +173,37 @@ Here is a guide for a quick setup of your `values-custom.yaml` file.
...
 
+>**Note:** The compatibility with older configurations is kept. The `host` and `port` values can be set to your MongoDB server hostname and port according to your setup.
 
->**Note:** Other custom connection options are supported, see MongoDB offcial documentation [here](https://docs.mongodb.com/v4.0/reference/connection-string/#connection-string-options).
+>**Note:** Other custom connection options are supported, see MongoDB connection string options offcial documentation [here](https://docs.mongodb.com/v4.0/reference/connection-string/#connection-string-options).
 
 ##### Authentication
 
-Depending on your mongoDB setup you must specify if an authentication is required or not by setting `usePassword` to `true` of `false`
+Depending on your mongoDB setup you must specify if an authentication is required or not. There is two options:
```
Comment (FROD-Neotys) [inline README.md:203]: Typo: "There is two options" -> "There are two options"

File: README.md [hunk 2]
```diff
@@ -173,32 +173,37 @@ Here is a guide for a quick setup of your `values-custom.yaml` file.
...
 
->**Note:** For MongoDB as a cluster of machines (replica set), you can specify the MongoDB URL of your cluster changing the `host` property value. You must also set `port` to `0`.
-*Example:* The MONGODB_HOST value must look like: 
-`rs1.mongo.mycompany.com:27017,rs2.mongo.mycompany.com:27017,rs3.mongo.mycompany.com:27017/admin`
+>**Example:** For MongoDB connection with DNS Seedlist Connection Format, the `host` value must look like: `mongodb+srv://rs1.mongo.mycompany.com:27017,rs2.mongo.mycompany.com:27017,rs3.mongo.mycompany.com:27017/admin`
 
+>**Note:** The compatibility with older configurations is kept. The `host` and `port` values can be set to your MongoDB server hostname and port according to your setup.
 
->**Note:** Other custom connection options are supported, see MongoDB offcial documentation [here](https://docs.mongodb.com/v4.0/reference/connection-string/#connection-string-options).
+>**Note:** Other custom connection options are supported, see MongoDB connection string options offcial documentation [here](https://docs.mongodb.com/v4.0/reference/connection-string/#connection-string-options).
```
Comment (FROD-Neotys) [inline README.md:199]: Typo: "official"

File: README.md [hunk 3]
```diff
@@ -173,32 +173,37 @@ Here is a guide for a quick setup of your `values-custom.yaml` file.
 
 ##### Host and port
 
-You need to replace `YOUR_MONGODB_HOST_URL` by your MongoDB server host.
-You also need to change the `port` value according to your MongoDB setup. 
+The preferred way to configure MongoDB connection information is to replace `YOUR_MONGODB_HOST_URL` by the
+full connection string URI. See [MongoDB official documentation](https://docs.mongodb.com/v4.0/reference/connection-string/). <br/>
+You must also set `port` to `0`. <br/>
+Username and/or password can be provided in the URI, special characters must be URL-encoded. By doing this, you must set `usePassword` to `false` in the below section "Authentication".
```
Comment (FROD-Neotys) [inline README.md:179]: Because of this, I think we need to store the mongo host as a cluster secret, just like the username and password.

**Review by FROD-Neotys — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (guillaumebert) [reply README.md:179]: Yes, we could add the URL to mongo secrets

**Pull request merged**